script.js

```diff
@@ -0,0 +1,322 @@
+  {
+    title: 'Tonic',
+    details: [
+      'CANOPY',
+      'Back End Dev',
+      '2015',
+    ],
+    info: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    tech: [
+      './images/Tag.png',
+      './images/Tag (1).png',
+      './images/Tag(2).png',
+    ],
+    src: 'img1',
+    url: [
+      './images/SnapshootPortfolio.png',
+      './imagefile/SnapshootPortfolio.png',
+    ],
+    buttons: [
+      [
+        ' https://michaelkith.github.io/portifolio1/',
+        'https://github.com/MICHAELKITH/desktop-mobile-version',
+      ],
+      [
+        './imagefile/Live_btn.png',
+        './imagefile/Source_Btn.png',
+      ],
+    ],
+  },
+  {
+    title: 'Multi Post Stories',
+    details: [
+      'Facebook',
+      'Full Stack Dev',
+      '2015',
+    ],
+    info: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    tech: [
+      './images/Tag.png',
+      './images/Tag (1).png',
+      './images/Tag(2).png',
+      './imagefile/Tag.png',
+    ],
+    src: 'img2',
+    url: [
+      './images/SnapshootPortfolio-2.png',
+      './imagefile/SnapshootPortfolio-1.png',
+    ],
+    buttons: [
+      [
+        ' https://michaelkith.github.io/portifolio1/',
+        'https://github.com/MICHAELKITH/desktop-mobile-version',
+      ],
+      [
+        './imagefile/Live_btn.png',
+        './imagefile/Source_Btn.png',
+      ],
+    ],
+  },
+  {
+    title: 'Facebook 360',
+    details:
+      [
+        'Facebook',
+        'Full Stack Dev',
+        '2015',
+      ],
+    info: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    tech: [
+      './images/Tag.png',
+      './images/Tag (1).png',
+      './images/Tag(2).png',
+      './imagefile/Tag.png',
+    ],
+    src: 'img3',
+    url: [
+      './images/SnapshootPortfolio-1.png',
+      './imagefile/SnapshootPortfolio-2.png',
+    ],
+    buttons: [
+      [
+        ' https://michaelkith.github.io/portifolio1/',
+        'https://github.com/MICHAELKITH/desktop-mobile-version',
+      ],
+      [
+        './imagefile/Live_btn.png',
+        './imagefile/Source_Btn.png',
+      ],
+    ],
+  },
+  {
+    title: 'Uber Navigation',
+    details: [
+      'UBER',
+      'Lead Developer',
+      '2015',
+    ],
+    info: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    tech: [
+      './images/Tag.png',
+      './images/Tag (1).png',
+      './images/Tag(2).png',
+      './imagefile/Tag.png',
+    ],
+    src: 'img4',
+    url: [
+      './images/SnapshootPortfolio-3.png',
+      './imagefile/SnapshootPortfolio-3.png',
+    ],
+    buttons: [
+      [
+        ' https://michaelkith.github.io/portifolio1/',
+        'https://github.com/MICHAELKITH/desktop-mobile-version',
+      ],
+      [
+        './imagefile/Live_btn.png',
+        './imagefile/Source_Btn.png',
+      ],
+    ],
+  },
+];
+
+// Hamburger
```

Comment:
- [ ] [Optional] Everything is good well done, but there are small changes you need to make please do so when I click the pop-up buttons it appears but when I scroll it then It scrolls all the way down to the contact form. Please fix it.
### Error
![when](https://user-images.githubusercontent.com/81727988/216076001-d8cb648b-8f71-452d-b129-8b43ff2e0f18.png)